Makefile

```diff
@@ -31,6 +31,18 @@ all: webserver
 webserver: $(SRC_FILES)
 	g++ $(CXXFLAGS) -I. $(SRC_FILES) -o webserver $(BOOST_FLAGS)
 
+docker:
+	rm -f deploy/webserver
+	docker build -t httpserver.build .
+	docker run httpserver.build > deploy/binary.tar
+	cd deploy && tar -xvf binary.tar
+	rm -f deploy/binary.tar
+	docker build -t httpserver deploy
+	docker run --rm -t -p 8080:8080 httpserver
```

Comment:
Maybe we should have different targets for building the docker and deploying it?